docs/extending-documentation.md

```diff
@@ -210,76 +262,54 @@ module.exports = {
 };
 ```
 
-In the code above we have 2 places in where you can do overrides: `./.storybook/all.override.md` for generic overrides and via `getLocalOverridePath` for each component, when needed, the `rel` needs to be the same in `lion` and your own project to be able to override the right file.
-
-We insert a remarkExtend plugin instance for both the local overrides and the global overrides, and pass the necessary arguments. Then we return the new array of plugins for Mdjs.
-
-In each file you need to add a specifier which section you want to override. As a first step we want to load `lea-tab` and `lea-tab-panel` also in the project. We need to add some imports to the top of the file.
+In the code example mentioned, we have 2 places in where you can do overrides: `./.storybook/all.override.md` for generic overrides and via `getLocalOverridePath` for each component. When needed, the `rel` needs to be the same in `lion` and our own project to be able to override the right file.
```

Comment:
Minor detail: numbers below 20 should ideally be written as text so (`two`) -> https://onzetaal.nl/taaladvies/getallen-in-letters-of-cijfers